package.json

```diff
@@ -38,6 +38,7 @@
     "clsx": "^2.1.1",
     "isbot": "^4.4.0",
     "lucide-react": "^0.408.0",
+    "pnpm": "^9.6.0",
```

Comment:
Remove pnpm from the project. You're supposed to install it on your laptop, not in the project

Reply:
alright ma